spec/domain/importers/content_details_spec.rb

```diff
@@ -33,6 +38,12 @@
       expect(latest_dimension_item.reload.number_of_word_files).to eq 94
     end
 
+    it 'stores the given quality metrics' do
+      subject.run
+      expect(latest_dimension_item.reload.number_of_spelling_errors).to eq 3
+      expect(latest_dimension_item.reload.number_of_readability_issues).to eq 10
+    end
+
```

Comment:
We also need to be sure that if an exception is thrown the exception is propagated, no data is persisted so the job is retried.